internal/install/command.go

```diff
@@ -55,6 +57,11 @@ var Command = &cobra.Command{
 
 			// Run the install.
 			if err := i.Install(); err != nil {
+				if err == types.ErrInterrupt {
+					fmt.Println("DS:DNFLKSDJFNLKJSDNFLJSKDNFKLJSDNFKLJNFLJSKDNFKJLn")
```

Comment:
🤣

Reply:
what?

Reply:
I don't think `fmt.Println("DS:DNFLKSDJFNLKJSDNFLJSKDNFKLJSDNFKLJNFLJSKDNFKJLn")` is needed here 😉